library/spdm_requester_lib/libspdm_req_common.c

```diff
@@ -24,11 +24,11 @@ uint16_t libspdm_allocate_req_session_id(libspdm_context_t *spdm_context)
     return (INVALID_SESSION_ID & 0xFFFF);
 }
 
-libspdm_return_t libspdm_build_opaque_data_supported_version_data(libspdm_context_t *spdm_context,
-                                                                  size_t *data_out_size,
-                                                                  void *data_out)
+void libspdm_build_opaque_data_supported_version_data(libspdm_context_t *spdm_context,
+                                                      size_t *data_out_size,
+                                                      void *data_out)
 {
-    size_t final_data_size;
+    size_t final_data_size = libspdm_get_opaque_data_supported_version_data_size(spdm_context);
```

Comment:
I don't recommend we do assignment in definition.

Can we move it just before ASSERT?

```
final_data_size = libspdm_get_opaque_data_supported_version_data_size(spdm_context);
LIBSPDM_ASSERT(*data_out_size >= final_data_size);
```

Reply:
Will do.